[2025-06-20T13:09:14.982+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-20T13:09:15.111+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_transform_to_dwh.load_dim_match manual__2025-06-20T13:09:10.150088+00:00 [queued]>
[2025-06-20T13:09:15.163+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_transform_to_dwh.load_dim_match manual__2025-06-20T13:09:10.150088+00:00 [queued]>
[2025-06-20T13:09:15.167+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-20T13:09:15.320+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_dim_match> on 2025-06-20 13:09:10.150088+00:00
[2025-06-20T13:09:15.341+0000] {standard_task_runner.py:63} INFO - Started process 1462 to run task
[2025-06-20T13:09:15.356+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_transform_to_dwh', 'load_dim_match', 'manual__2025-06-20T13:09:10.150088+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpzobpau3p']
[2025-06-20T13:09:15.363+0000] {standard_task_runner.py:91} INFO - Job 224: Subtask load_dim_match
[2025-06-20T13:09:15.634+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_transform_to_dwh.load_dim_match manual__2025-06-20T13:09:10.150088+00:00 [running]> on host 5ea278fdf3a6
[2025-06-20T13:09:16.051+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_transform_to_dwh' AIRFLOW_CTX_TASK_ID='load_dim_match' AIRFLOW_CTX_EXECUTION_DATE='2025-06-20T13:09:10.150088+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-20T13:09:10.150088+00:00'
[2025-06-20T13:09:16.054+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-20T13:09:16.091+0000] {etl_pipeline.py:451} INFO - Ensuring dimension table 'dimmatch' exists...
[2025-06-20T13:09:16.094+0000] {etl_pipeline.py:453} INFO - Loading/updating data into 'dimmatch'...
[2025-06-20T13:09:16.116+0000] {etl_pipeline.py:455} INFO - Operation complete. 1321 new rows inserted into dimmatch.
[2025-06-20T13:09:16.154+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-20T13:09:16.157+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-20T13:09:16.170+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nba_transform_to_dwh, task_id=load_dim_match, run_id=manual__2025-06-20T13:09:10.150088+00:00, execution_date=20250620T130910, start_date=20250620T130915, end_date=20250620T130916
[2025-06-20T13:09:16.255+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-20T13:09:16.284+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-20T13:09:16.311+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
